src/panels/TcpDumpPanel.ts

```diff
@@ -28,7 +33,7 @@ function getTcpDumpCommand(capture: string): string {
 function getCaptureFromCommand(command: string, commandWithArgs: string): string | null {
     if (command !== "tcpdump") return null;
     if (!commandWithArgs.startsWith(tcpDumpCommandBase)) return null;
-    const fileMatch = commandWithArgs.match(new RegExp(`\-w ${captureFilePathRegex}`));
+    const fileMatch = commandWithArgs.match(new RegExp(`\\-w ${captureFilePathRegex}`));
```

Comment:
Escaping the `\` so that a `\` character appears in the regex string to capture a literal `-` in the command. I haven't quite thought through how this could have been correct before, so it's worth doing a bit more testing (although it does seem to work now).

Reply:
Turns out that the `-` character [doesn't always need escaping](https://stackoverflow.com/a/5484178). In this case it wasn't being escaped (because the escape character `\` itself needed escaping to appear in the regex string), but it still worked because this was one of the cases where it didn't need escaping (it was correctly interpreted as a literal).

This change escapes it, which is also valid.